.github/workflows/wolfictl-lint.yaml

```diff
@@ -0,0 +1,17 @@
+name: Wolfictl Lint
+
+on:
+  pull_request:
+    branches: ['main']
+jobs:
+  lint:
+    name: Wolfictl Lint
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: wolfi-dev/wolfictl/.github/actions@main
```

Comment:
This is using main for now to help speed up bringing in new linting rules early on. In the future lets switch to a released wolfi binary or image